src/stored-message.ts

```diff
@@ -1,8 +1,12 @@
 import {DefaultLogFields} from 'simple-git'
 import {nullCommit} from './commit'
 
-export const CREATE_JOB_SUBJECT_PREFIX = 'CLAIM LOCK: JOB: '
-export const MARK_JOB_AS_DONE_SUBJECT_PREFIX = 'RELEASE LOCK: JOB DONE: '
+export const NEW_JOB_SUBJECT_PREFIX = '📝🈺: '
+export const FINISHED_JOB_SUBJECT_PREFIX = '📝✅: '
+
+// TODO: Code Review: should be only use our own class Commit?
+// We are exposing the class Commit in the main module and using DefaultLogFields internally.
+// We can keep doing that but we should rename "nullCommit()" function to "emptyDefaultLogFields()".
```

Comment:
It actually is somehow confusing the conceptual relation between "commit" and "LogFields". Are they the same? if not, what is the difference between a commit and logFields?. Also I don't like at all that last name, is very generic and inexpressive.